src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -533,12 +533,12 @@ protected void RemeasureLayout(VisualElement formsElement)
 			if (IsHorizontal)
 			{
 				var request = formsElement.Measure(double.PositiveInfinity, CollectionView.Frame.Height);
-				Controls.Compatibility.Layout.LayoutChildIntoBoundingRegion(formsElement, new Rect(0, 0, request.Width, CollectionView.Frame.Height));
+				formsElement.Arrange(new Rect(0, 0, request.Width, CollectionView.Frame.Height));
 			}
 			else
 			{
 				var request = formsElement.Measure(CollectionView.Frame.Width, double.PositiveInfinity);
-				Controls.Compatibility.Layout.LayoutChildIntoBoundingRegion(formsElement, new Rect(0, 0, CollectionView.Frame.Width, request.Height));
+				formsElement.Arrange(new Rect(0, 0, CollectionView.Frame.Width, request.Height));
```

Comment:
`LayoutChildIntoBoundingRegion` already would just short circuit and call "Arrange" so these changes shouldn't really move the cheese at all

Reply:
But this will call arrange and not layout ? do we need both ?

Reply:
No, the code inside "LayoutChildIntoBoundingRegion" only calls "Layout" if the handler isn't attached. 

<img width="871" alt="image" src="https://github.com/user-attachments/assets/872d5f31-2bcd-4b76-85dd-778295a9840d">


All that the Layout call does is set the Bounds. Which is kind of a noop as far as MAUI

<img width="521" alt="image" src="https://github.com/user-attachments/assets/19cabb32-7ff4-4845-8895-3470dcaf6820">

Plus the Arrange call will already set the "Frame" to the value passed in 

<img width="468" alt="image" src="https://github.com/user-attachments/assets/2dd44ea7-4a1b-4436-93d8-05c582a4b034">

Which is basically the same thing as setting the Bounds. 

So, calling (IView.Arrange) basically already calls "Arrange" on the Handler and then does the equivalent of calling Layout